src/utilities/sequence-editor/from-seq-json.ts

```diff
@@ -115,30 +115,24 @@ function seqJsonVariableToSequence(
   variables: [VariableDeclaration, ...VariableDeclaration[]],
   type: 'INPUT_PARAMS' | 'LOCALS',
 ): string {
-  let sequence = `@${type}`;
-
-  if (type === 'INPUT_PARAMS') {
-    variables.forEach(variable => {
-      sequence += ` ${variable.name} `;
-
-      if (Object.keys(variable).length > 1) {
-        sequence += '{ ';
-
-        for (const key of Object.keys(variable)) {
-          if (key !== 'name') {
-            sequence += `"${key}": "${variable[key]}", `;
-          }
-        }
-
-        // Remove the trailing space and commma from the last property.
-        sequence = `${sequence.substring(0, sequence.length - 2)} }`;
-      }
-    });
-  } else {
-    sequence += ` ${variables.map(variable => variable.name).join(' ')}`;
-  }
+  let sequence = `@${type}_BEGIN\n`;
+
+  sequence += variables
+    .map(variable => {
+      const name = variable.name;
+      const type = variable.type ? ` ${variable.type}` : '';
+      const enumName = variable.enum_name ? ' ' + variable.enum_name : '';
```

Comment:
can we template string this following line as well?

Reply:
Sure it is addressed in this commit:

236a51139bc220cb1019233f607931e1185b8539